components/skeleton/demo/skeleton-test-typography.js

```diff
@@ -0,0 +1,67 @@
+import {
+	bodyCompactStyles,
+	bodySmallStyles,
+	bodyStandardStyles,
+	heading1Styles,
+	heading2Styles,
+	heading3Styles,
+	heading4Styles,
+	labelStyles
+} from '../../typography/styles.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { linkStyles } from '../../link/link.js';
+import { SkeletonMixin } from '../skeleton-mixin.js';
+
+export class SkeletonTestTypography extends SkeletonMixin(LitElement) {
+
+	static get styles() {
+		return [
+			super.styles,
+			bodyStandardStyles,
+			bodyCompactStyles,
+			bodySmallStyles,
+			heading1Styles,
+			heading2Styles,
+			heading3Styles,
+			heading4Styles,
+			labelStyles,
+			linkStyles,
+			css`
+			:host {
+				display: block;
+			}
+			.box {
+				border: 1px solid black;
+				height: 100px;
+				width: 300px;
+			}
+		`];
+	}
+
+	render() {
+		return html`
+			<div class="box d2l-skeletize"></div>
+			<hr>
+			<p class="d2l-body-standard d2l-skeletize d2l-skeletize-20">Body Standard</p>
+			<p class="d2l-body-compact d2l-skeletize d2l-skeletize-15">Body Compact</p>
+			<p class="d2l-body-small d2l-skeletize d2l-skeletize-10">Body Small</p>
+			<p class="d2l-label-text d2l-skeletize d2l-skeletize-5">Label</p>
+			<hr>
+			<p class="d2l-body-standard d2l-skeletize-lines-3">Line 1<br>Line 2<br>Line 3</p>
+			<p class="d2l-body-compact d2l-skeletize-lines-3">Line 1<br>Line 2<br>Line 3</p>
+			<hr>
+			<a href="https://d2l.com" class="d2l-link d2l-skeletize">Link (Normal)</a><br>
+			<a href="https://d2l.com" class="d2l-link d2l-link-main d2l-skeletize">Link (Main)</a><br>
+			<a href="https://d2l.com" class="d2l-link d2l-link-small d2l-skeletize">Link (Small)</a>
+			<hr>
+			<h1 class="d2l-heading-1 d2l-skeletize d2l-skeletize-25">Heading 1</h1>
+			<h2 class="d2l-heading-2 d2l-skeletize d2l-skeletize-20">Heading 2</h2>
+			<h3 class="d2l-heading-3 d2l-skeletize d2l-skeletize-15">Heading 3</h3>
+			<h4 class="d2l-heading-4 d2l-skeletize d2l-skeletize-10">Heading 4</h4>
+			<hr>
+			<h3 class="d2l-heading-3 d2l-skeletize d2l-skeletize-15">Heading 3<br>spanning multiple lines</h3>
```

Comment:
This component looks like this:

![Screen Shot 2020-09-16 at 6 16 22 PM](https://user-images.githubusercontent.com/5491151/93398017-c1bdda00-f848-11ea-89bd-18e13230aab9.png)

And in skeleton-mode:

![Screen Shot 2020-09-16 at 6 16 51 PM](https://user-images.githubusercontent.com/5491151/93398057-d00bf600-f848-11ea-8015-2b55bd3e7ae5.png)